tests/integration/extenders/MailTest.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * For detailed copyright and license information, please view the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace Flarum\Tests\integration\extenders;
+
+use Flarum\Extend;
+use Flarum\Mail\DriverInterface;
+use Flarum\Settings\SettingsRepositoryInterface;
+use Flarum\Tests\integration\RetrievesAuthorizedUsers;
+use Flarum\Tests\integration\AuthenticatedTestCase;
+use Illuminate\Contracts\Validation\Factory;
+use Illuminate\Support\MessageBag;
+use Swift_NullTransport;
+use Swift_Transport;
+
+class MailTest extends AuthenticatedTestCase
```

Comment:
This base class does not exist anymore. Please rebase on the latest master and change accordingly.

Reply:
Sorry got sidetracked with my categories extension, I'll have all my extender PRs rebased sometime today

Reply:
Done!